internal/resources/grafana/resource_sso_settings_test.go

```diff
@@ -0,0 +1,292 @@
+package grafana_test
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/grafana/grafana-openapi-client-go/client"
+	"github.com/grafana/grafana-openapi-client-go/models"
+
+	"github.com/grafana/terraform-provider-grafana/internal/common"
+	"github.com/grafana/terraform-provider-grafana/internal/testutils"
+)
+
+func TestSSOSettings_basic(t *testing.T) {
+	testutils.CheckOSSTestsEnabled(t, ">=10.4.0")
```

Comment:
```suggestion
	testutils.CheckCloudInstanceTestsEnabled(t)
```

We should be able to test this on Grafana Cloud in the meantime, right?

Reply:
The new API will be available in cloud next week the soonest. However it has to be enabled from the feature flags, it won't be available by default.

Reply:
That's fine, we can enable flags on our test instance. I did it for nested folders. We can do it as a follow-up once it's deployed in cloud though